lms/templates/user_dropdown.html

```diff
@@ -20,7 +20,7 @@
             <img class="menu-image" src="${profile_image_url}" alt="">
             ${username}
         </a>
-        <div role="group" aria-label="User menu" class="user-menu">
+        <div role="group" aria-label="User menu">
```

Comment:
@schenedx the bug on Andy's sandbox is that the user dropdown on pages that use the pattern library is significantly misaligned.

![](https://i.bjacobel.com/20160913-7a5p1.png)